packages/documentation/src/pages/getting-started/common-tasks/updating-formation.mdx

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
It looks like this doc replaces the content in the README. Can we update the README to point to this page for how to make changes to formation/formation-react, so we don't have duplicate, out of date content in there?